plugin/etcd/README.md

```diff
@@ -107,6 +113,9 @@ etcd skydns.local {
 ...
 ~~~
 
+Before getting started with examples, please setup `etcdctl` (with `etcdv3` API) as explained [here](https://coreos.com/etcd/docs/latest/dev-guide/interacting_v3.html). This will help you put sample keys to you etcd server easily.
```

Comment:
with *these*

Reply:
"to you etcd" -> you to put sample keys in your etcd server.

Reply:
done